src/spring-cloud/azext_spring_cloud/tests/latest/test_asc_validator.py

```diff
@@ -121,14 +121,14 @@ def test_lack_of_parameters(self):
                        reserved_cidr_range='10.0.0.0/14', sku=None, location='eastus')
         with self.assertRaises(CLIError) as context:
             validate_vnet_required_parameters(ns)
-            self.assertEqual('--app-subnet, --service-runtime-subnet must be set when deploying to VNet',
-                             str(context.exception))
+        self.assertEqual('--app-subnet, --service-runtime-subnet must be set when deploying to VNet',
+                         str(context.exception))
```

Comment:
What happens if assertEqual fails in the original code?

Reply:
in the original code, the assertEqual will never be executed since `validate_vnet_required_parameters` will throw exception and be caught by `self.assertRaises(CLIError)`